test/azure/AcceptanceTests/asynctests/test_lro.py

```diff
@@ -494,19 +499,20 @@ async def test_polling_interval_config(self, cookie_policy, credential, client):
         ]
         client_one_second = AutoRestLongRunningOperationTestService(credential, base_url="http://localhost:3000", policies=policies, polling_interval=1)
         one_second_polling_interval_start_time = time.time()
-        product2 = await client_one_second.lros.post_double_headers_final_azure_header_get_default()
+        poller = await client_one_second.lros.begin_post_double_headers_final_azure_header_get_default()
+        product2 = await poller.result()
         one_second_polling_interval_duration = time.time() - one_second_polling_interval_start_time
         assert abs(one_second_polling_interval_duration - 1) < 0.1
         assert product1 == product2
 
     @pytest.mark.asyncio
     async def test_passing_kwargs(self, client, product):
-        process = await self.lro_result(client.lros.put200_succeeded, product, content_type="application/json")
+        process = await self.lro_result(client.lros.begin_put200_succeeded, product, content_type="application/json")
         assert "Succeeded" == process.provisioning_state
 
     @pytest.mark.asyncio
     async def test_lro_list(self, client, product):
-        products = await self.lro_result(client.lros.post202_list)
+        products = await self.lro_result(client.lros.begin_post202_list)
```

Comment:
Can you add a test for continuation token in the sync acceptance test?

Reply:
I did a test already called "test_post_double_headers_final_continuation_token" am I missing something?

Reply:
apologies, I didn't see the sync test_lro.py file